src/controllers/adminStatistics.ts

```diff
@@ -0,0 +1,64 @@
+import { Request, Response } from "express";
+import { sendResponse } from "../utils/http.exception";
+import jwt, { JwtPayload } from "jsonwebtoken";
+import { User } from "../database/models/User";
+import { Role } from "../database/models/role";
+import { Sales } from "../database/models/sales";
+import { Product } from "../database/models/product";
+
+export const adminStatistics = async (req: Request, res: Response) => {
+	try {
+		const admin_token = req.headers.authorization?.split(" ")[1];
+		if (admin_token) {

```

Comment:
As long as we have middleware for checking if user is admin, we don't need to use these checking in controller.
just remove it and middleware will do the job.